app/src/main/java/ro/code4/monitorizarevot/ui/main/MainActivity.kt

```diff
@@ -64,14 +66,23 @@ class MainActivity : BaseActivity<MainViewModel>() {
             }
         }
         navLogout.setOnClickListener {
-            // TODO logout
+            logoutUser()
         }
-
     }
 
-
     override fun onSupportNavigateUp(): Boolean {
         return navController.navigateUp(appBarConfiguration) || super.onSupportNavigateUp()
     }
 
+    private fun logoutUser() {
+        viewModel.logout()
+        openLogin()
```

Comment:
The `logout()` may change in the future. Normally you should observe a `SingleLiveEvent` in the activity when the logout finished (e.g. a call in backend to logout and invalidate token might not be instant or might fail) and redirect user to login once you're sure it completed successfully.